Mobile Buy SDK/Mobile Buy SDK/Models/Persistent/BUYProduct.h

```diff
@@ -52,6 +52,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (NSArray<BUYProductVariant *> *)variantsArray;
 
+/**
+ * Fetch the prices from variants and cache the lowest price on minimumPrice.
+ */
+- (void)updateMinimumPrice;
```

Comment:
No, that would be very inefficient and expensive. We need to set it as soon as the variants as set. Lazy calculation would force us to fetch all the variants immediately, which we don't want to do when we're displaying a list of products, for example.